File: src/main/java/no/novari/flyt/egrunnerverv/gateway/dispatch/repository/InstanceReceiptDispatchRepository.java
package no.novari.flyt.egrunnerverv.gateway.dispatch.repository;

import no.novari.flyt.egrunnerverv.gateway.dispatch.model.InstanceReceiptDispatchEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface InstanceReceiptDispatchRepository extends JpaRepository<InstanceReceiptDispatchEntity, String> {
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/dispatch/repository/InstanceReceiptDispatchRepository.kt
package no.novari.flyt.egrunnerverv.gateway.dispatch.repository

import no.novari.flyt.egrunnerverv.gateway.dispatch.model.InstanceReceiptDispatchEntity
import org.springframework.data.jpa.repository.JpaRepository

interface InstanceReceiptDispatchRepository : JpaRepository<InstanceReceiptDispatchEntity, String>
